packages/Counter/src/Counter.styles.js

```diff
@@ -18,31 +18,45 @@ const counterSizeStyles = {
   `,
 };
 
-const counterTypeStyles = {
-  default: `
+const counterColorStyles = {
+  grey: `
     background: ${tokens.color.blackLighten80};
     border-radius: ${tokens.border.radius};
     border: 1px solid ${tokens.border.color};
     color: ${tokens.textColor.subtle};
   `,
 
   red: `
-    border-radius: ${tokens.counter.radius};
+    border-radius: 50px;
```

Comment:
Couldn't find any other way to get it to look like what it was before. So keeping the 50px as just special case here (without a token) for the red / blue counters.